dashboard/app/models/assessment_activity.rb

```diff
@@ -6,7 +6,7 @@
 #  user_id         :integer          not null
 #  level_id        :integer          not null
 #  script_id       :integer          not null
-#  level_source_id :integer
+#  level_source_id :integer          unsigned
```

Comment:
In the `dashboard` directory, I think you can run `bundle exec rake annotate_models` (after running the migration) to fix this.

Reply:
Per offline discussion with @suresh, tried this but no luck. Actually was able to fix this (and other tables with `bigint` columns) by upgrading the annotate gem to 2.7.5, but decided that was out of scope for this work. Leaving as-is for now.